react-native.js

```diff
@@ -42,10 +42,10 @@ module.exports = {
     {
       files: ['*.js', '*.jsx'],
       settings: {
-        'import/extensions': [...extensions.JS, ...extensions.JS_REACT_NATIVE],
+        'import/extensions': extensions.ALL,
         'import/resolver': {
           node: {
-            extensions: [...extensions.JS, ...extensions.JS_REACT_NATIVE],
+            extensions: extensions.ALL,
```

Comment:
Looks like this is the same as in node preset, which we extend from, so it's not really necessary to override anything, isn't it?

Reply:
I've removed the overrides in react-native.js, and it seems to work the same in my sample project. Do you have any ideas how to test it further?

Reply:
Now that I look at it, I think that `TS` extensions should feature `'.js', '.jsx'` as well. Now we let Node resolve `.native.js`, `.android.js` and `.ios.js`, which is not what we want. Will update later, unless you want to tackle it :)